packages/core/src/lib/utils/providers.ts

```diff
@@ -75,12 +77,20 @@ function normalizeOAuth(
     c.redirectProxyUrl = `${c.redirectProxyUrl}/callback/${c.id}`
   }
 
+  let discoveredAs: o.AuthorizationServer | undefined
+  if (c.issuer) {
+    const issuer = new URL(c.issuer)
+    const discoveryResponse = await o.discoveryRequest(issuer)
+    discoveredAs = await o.processDiscoveryResponse(issuer, discoveryResponse)
+  }
```

Comment:
- Not working if authorization server does not follow the specifications defined in OpenID Connect Discovery 1.0
- Customized wellKnown is not available